### DISCUSSION TITLE
OneDrive sync - PC to mobile setup

### DISCUSSION THREAD

**Discussion by Mayome** [remotely-save/remotely-save#428] [Show and tell]
Ok look, I've seen a lot of ppl having issues with this, I made it work and wrote it down so i wouldn't forget, might as well post it here. Hopefully it helps someone!

### step 1: PC to OneDrive
1) install `remotely-save` on your PC obsidian install (follow the plugin instructions for OneDrive sync)
2) sync manually in Obsidian (left sidebar, circling arrow icon)
3)  note where it creates the sync folder (default: `OneDrive\Apps\remotely-save`) and your vault names, check if they synced there correctly
### step 2: mobile to OneDrive
1) open obsidian on your mobile device, go to your existing vault, install `Remotely Save` there as well, follow the instructions for OneDrive sync
2) **NOTE: every vault on iOS will need its own  `Remotely Save` plugin install!!!** (not tested on android)
3) sync manually, check if the mobile vault shows up on OneDrive
4) if not, i don't know how to help you. ask your preferred deity and/or post a comment!
### step 3: OneDrive to mobile
1) for your PC vaults to sync to your mobile device, *you need to make an empty vault of the same name on your mobile device*. so if my PC vault is called "obsidian personal", i make a new vault on mobile called "obsidian personal".
2) **DO NOT MAKE ANY FILES OR FOLDERS THERE**. it will probably give you sync issues ~~but i didn't check honestly~~
3) go directly to vault settings. download and setup `Remotely Save` again for this vault. set it up same as before
4) sync manually
5) your PC vault files should appear now!!!! if they didnt - see step 2 point 4.

and that's it really! lmk if anything is unclear or incorrect, but it did work for me, so you know. 🤷

**Comment by fyears** [comment]
thx!

actually i still don't know where the doc is not clear for users. of course since i am the author i have blind spots. is it that the "keep folder empty" or "keep vault names the same" not sure clear in the beginning?

**Reply by Mayome** [reply]
Hey hey, do forgive me if I duplicated an existing doc/tutorial, but i genuinely could not find the OneDrive-to-mobile (bidirectional) setup doc, and it wasn't clear for me in the instructions in the plugin settings! I might've not looked hard enough, which is fair : D but I figured that if I couldn't find it, I might be not the only one ahahah. i worked backwards from #382 and just put what i did as simply as possible

**Comment by futurefowl** [comment]
Thanks for this. I may have totally missed it too, but I was struggling with this same thing. These steps really helped me get everything working.

**Comment by vincentburkhardt** [comment]
Really helpful, was looking exactly for this.
Thanks a lot (:
